### ISSUE TITLE
Command on shutdown, also can't sign up to forum

### ISSUE THREAD

**Issue by theplasticbling** [EventGhost/EventGhost#344]
First off I can't sign up to the forum. I can't get an activation email. Tried contacting moderators with no luck. I'm trying to sign up to see if anybody knows a way to activate a command on shutdown whilst using a custom shell. I've got commands working when using explorer.exe, but when using another shell I can't figure out a command that works. Anybody know of a workaround? Thanks

**Comment by GruberMarkus** [comment]
Same here regarding activation e-mail since about a week. I contacted the administrator of the forum via the contact form on the homepage yesterday and await his answer (let's hope he e-mail problems are outbound only ...).

**Comment by kdschlosser** [comment]
All e-mails from the domain are not functioning at the moment. DNS/Name Server related issue. 

I am the site administrator. if you give me your login names for the forum I will send an e-mail to the address you registered with (to confirm the account does belong to you) after you respond I will then activate your account.

**Comment by kdschlosser** [comment]
I just added a post to the announcements on the forums telling people of the issue with registrations. and directed them to here to go through this same process.

**Comment by theplasticbling** [comment]
My username is theplasticbling. As for the command on shutdown, I'd like to turn my monitor off when I shutdown my PC. The commands on the bottom of the first page of this thread work: http://www.eventghost.net/forum/viewtopic.php?t=1230

Problem is, they only work if I am using the stock windows shell (explorer.exe). When I use a different shell (my PC is set up as a gaming machine so I automatically run Steam at startup rather than explorer.exe), the commands do not work. I'm not looking for commands to change shells, just a command that runs on shutdown when I'm not using the stock windows shell.

**Comment by kdschlosser** [comment]
they should work if you are using Windows explorer or not. Those events come from the core Windows notification system. which is located in the kernel32 dll and not in explorer.exe

Now that being said. is whatever it is you are using is breaking the notification system then there is not much I can do for ya. There is a good chance that not having a shell of any kind is what is breaking that notification system.

Now I have another silly question.. doesn't your monitor power down when it gets a loss of video signal??? I know mine does. which seems kind of odd that yours does not.

I am going to send that e-mail now. I have to run out for a bit and I will not be able to activate your account until this evening. I do apologize for that.

**Comment by theplasticbling** [comment]
Thanks for the info. I actually upgraded to an OLED TV and it doesn't automatically power off, my old monitor did.

**Comment by kdschlosser** [comment]
OK if you are using a TV you will not be able to power it off from EventGhost unless you specifically use a plugin made for the TV.

What is the brand and model of the TV?
How do you have the PC connected to the TV?? (HDMI, DVI, Display Port, VGA)

**Comment by theplasticbling** [comment]
>           OK if you are using a TV you will not be able to power it off from EventGhost unless you specifically use a plugin made for the TV.
> What is the brand and model of the TV?
> How do you have the PC connected to the TV?? (HDMI, DVI, Display Port, VGA)

I have an LG C6 55" connected through HDMI. Commands work fine to turn the tv on and off with USB-UIRT. Just doesn't work to turn off the TV when I tell the PC to shutdown unless the Windows shell is running. Do you know of a specific 'on shutdown' command I could try that EventGhost can recognize whilst explorer.exe isn't running?

**Comment by kdschlosser** [comment]
hmmm....

let me check and see if we even generate an event for shutting down.
if not that is something that should be added.

The only thing is IDK if there is a way to know if it is a shutdown or a restart. I think all that happens is a call gets made to EG to close. we don't get the actual reason why.

**Comment by theplasticbling** [comment]
>           hmmm....
> let me check and see if we even generate an event for shutting down.
> if not that is something that should be added.
> The only thing is IDK if there is a way to know if it is a shutdown or a restart. I think all that happens is a call gets made to EG to close. we don't get the actual reason why.

Did you find an event by chance?

**Comment by kdschlosser** [comment]
I have written some code that is supposed to do this.  me being the dingbat that I am forgot that this is already built into the core. and it grabs the event before an event registered after it. and EG gets shutdown before the plugin can do anything. So I have to either come up with a way to override that process or it has to be built into the core of EG. Having this in the cor is something I do feel should be added. But it is not something we would add to this next version. So I am tossing ideas around as to how to make it work as a plugin for the time being.

It can be done. It is a matter of getting the way to do it,

Also. There is no way to know if the system is shutting down or the system is restarting.

**Comment by kdschlosser** [comment]
here you go sir.... You asked and I delivered... plus I threw in a couple of bonuses. you will also get an event if the logged in user logs off. and you have actions to block the logoff and also block the shutdown. I hope this meets your expectations.


http://www.eventghost.net/forum/viewtopic.php?f=9&t=10276

**Comment by kdschlosser** [comment]
does that do what you wanted it to do?

**Comment by Ti-As** [comment]
Hello KD,
hopefully this is a way to get in contact with you. Registration for forum isn't possible and the contact sheet obviously doesn't wotk, too. So many questions ... Could you help?

**Comment by kdschlosser** [comment]
provide me the username you registered on the forum with and I will activate the account. Sometimes an e-mail provider filters out email addresses. and the contact form on the website is a catchall for mostly spam e-mails.

**Comment by ThomasBott** [comment]
... Or something with the SMTP settings is wrong. I was also wondering that no one registered in the forum for a long time, do I just tried to change my E-Mail address in the forum, what requires a re-activation via the newly registered E-mail address and I also didn't receive a mail. (No, it's not blocked by a spam filter, I just didn't get the mail and I'm 100% sure I entered the right E-mail address)
.. I'm locked out from the forum now btw. ^^°

**Comment by kdschlosser** [comment]
OK you are back on @ThomasBott 
I will look into the issue with the e-mail this evening to see what is going on.

**Comment by kdschlosser** [comment]
You are correct about the SMTP being outta wack. Google must have changed something because that is who I was bouncing off of to send the e-mails.

it is telling me bad credentials. I am going to check that now.

**Comment by kdschlosser** [comment]
well google fucked us. have to have some kind of a premium thing in order to use the gmail smtp relay service it appears.

**Comment by kdschlosser** [comment]
OK never mind I found the problem.

stupid gmail settings it automatically with turn on the blocking of less secure applications if it is not being used regularly. we do not get all that many new users so I am going to have to remind myself to check this setting from time to time

**Comment by Ti-As** [comment]
Did you get my mails? Sent two.

**Comment by Ti-As** [comment]
Did you get my mails? Sent two.

**Comment by kdschlosser** [comment]
where are you sending e-mails to? and why?

**Comment by Ti-As** [comment]
testing

**Comment by Ti-As** [comment]
I've received all these messages here by mail:

> where are you sending e-mails to? and why?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or unsubscribe.

I did reply directly! Adress: reply+AP6RPN553CXOLZF6JNFJL5V5CQNSBEVBNHHBOY5CM4@reply.github.com
Messages came from: notifications@github.com
Will test the "reply.github.com" again

I replied to give you additional forum account information.
Forum username should be the same like here. But please check email address, too. Should be this.is.xxxx@gmxxxx.com (-> Ti-As)
OR "firstname"."surname"@ but without vowels (-> "frstnm.srnam"@) without the quots. I have probably registered both.

**Comment by Ti-As** [comment]
No, i replied to notifications@github.com. If it was the "reply+..." address it would have been here asa comment.

**Comment by Ti-As** [comment]
This is replay by mail ...

**Comment by Ti-As** [comment]
Any news?

**Comment by Ti-As** [comment]
Have removed my comments, ThomasBott will contact you to delete any registered forum information so I can start from the scratch.
Thank you
